middle.mkd

```diff
@@ -331,14 +331,17 @@ height in meters above or below the WGS 84 reference ellipsoid. In the
 absence of elevation values, applications sensitive to height or depth
 SHOULD interpret positions as being at local ground or sea level.
 
-Other coordinate reference systems, including ones described by CRS
-objects of the kind defined in [GJ2008] are NOT RECOMMENDED. GeoJSON
+Other coordinate reference systems are NOT RECOMMENDED. GeoJSON
 processing software SHALL NOT be expected to have access to coordinate
-reference systems databases. Applications requiring a CRS other than the
-default MUST assume all responsibility for CRS identification,
-coordinate accuracy, and interpretation of missing elevation values.
-Furthermore, GeoJSON coordinates MUST NOT under any circumstances use
-latitude, longitude order.
+reference system databases or be expected to parse and interpret the CRS
+objects described in [GJ2008]. Furthermore, to prevent coordinate order
+confusion for parsers that do not have a CRS database, GeoJSON
+coordinates MUST NOT under any circumstances use latitude, longitude
+order. For example, GeoJSON data using a coordinate reference system
```

Comment:
@jyutzler Thanks for pointing that out.

@martinthomson the authors, mostly, see coordinate reversal as a different category of CRS confusion (as Jeff points out) from that of a datum shifts and want to eliminate the lat vs lon coordinate reversal category as much as possible. A scenario where implementers split on coordinate order and poorly supported the other side was easy to imagine and threats to bring it about came up a few times. It's a different category because it seemed to be an existential threat to the format. Is long/lat GeoJSON ubiquitous enough that we can drop the specific exclusion of lat/long CRS? It might be.

@adoyle excluding EPSG:4326 without naming it or providing an explanation feels too arbitrary to me.